portfolio/src/main/webapp/style.css

```diff
@@ -67,3 +72,50 @@ h1 {
   font: 10vh "Lucida Console", Monaco, Monospace;
 }
 
+#intro {
+  background-color: rgb(204, 124, 97);
+  display: inline-block;
+  padding: 10vh 15vw;
+}
+
+h2 {
+  font: bold 30px "Trebuchet MS", Helvetica, sans-serif;
+  margin-bottom: 15px;
+}
+
+#intro-hand {
+  float: left;
+  width: 15vw;
+}
+
+#intro-hand>p {
+  font-size: 7vw;
+  text-align: center;
+}
+
+#intro-hand>p:hover {
+  animation: wave 0.5s alternate infinite;
+  animation-timing-function: ease-in-out;
+}
+
+@keyframes wave {
+  0% {transform: rotate(-10deg);}
+  100% {transform: rotate(10deg);}
+}
+
+#intro-text {
+  color: whitesmoke;
+  float: right;
+  width: 55vw;
+}
+
+#intro-text>p {
+  color: gainsboro;
+  font: 15px Verdana, Geneva, sans-serif;
+  line-height: 1.5;
+}
+
+#intro-text>p:first-of-type {
+  margin-bottom: 30px;
+}
+
```

Comment:
nit: remove trailing blank line. Note this is different from having a 'newline' character at the end. I think your text editor might have an option to always ensure there's a newline character at the end.